pkg/security/module/cws.go

```diff
@@ -415,7 +415,9 @@ func (c *CWSConsumer) HandleEvent(event *model.Event) {
 	}
 
 	if ruleSet := c.GetRuleSet(); ruleSet != nil {
-		ruleSet.Evaluate(event)
+		if (event.AutoSuppressionEnabled && event.IsSecurityProfileFoundAndPresent()) || !ruleSet.Evaluate(event) {
+			ruleSet.PushDiscarders(event)
```

Comment:
nit : I would call it EvaluateDiscarders, PushDiscarders makes me think that it will acutally push a discarder to the kernel